index.html

```diff
@@ -10,7 +10,7 @@
     <title>My Portifolio</title>
 </head>
 <body>
-    
+    <!--Welcome Page Start-->
```

Comment:
A horizontal scroll seems to have been introduced on the page. Kindly modify the styles to avoid the horizontal scroll for a better user experience on the website.

![image](https://user-images.githubusercontent.com/20554583/192075599-b92e94ec-64c9-42ce-bcc2-1b79603f2b8a.png)